src/screens/shared/dashboard/StatCards.js

```diff
@@ -23,49 +36,183 @@ const styles = {
+    const totalHarvests = await getAllTotalHarvestsForStatCard();
+    const recentHarvests = await getAllRecentHarvestLogsForStatCard();
+    const KSFarms = farms.filter(farm => farm.ksAffiliated);
+    const KSGAP = [];
+    for (let i = 0; i < GAPCertification.length; i += 1) {
+      if ('ksAffiliated' in GAPCertification[i]) {
+        if (GAPCertification[i].ksAffiliated.includes(true)) {
+          KSGAP.push(GAPCertification[i]);
+        }
+      }
+    }
+
+    const numGAPCertified = GAPCertification.filter(farm => farm.gapCertified);
+
+    const numKSGAPCertified = KSGAP.filter(farm => farm.gapCertified);
+
+    const numFarmReferred = GAPCertification.filter(
+      farm => farm.farmReferred === 'Complete'
+    ).length;
+
+    const numKSGAPAccepted = KSGAP.filter(
+      farm => farm.farmAccepted === 'Complete'
+    ).length;
+
+    const numGAPApplied = GAPCertification.filter(
+      farm => farm.farmApplied === 'Complete'
+    );
+
+    const numKSGAPApplied = KSGAP.filter(
+      farm => farm.farmApplied === 'Complete'
+    );
+
+    const percentKSGAPCertified = Math.round(
+      (numKSGAPCertified.length / KSFarms.length) * 100
+    );
+    const percentKSGAPApplied = Math.round(
+      (numKSGAPApplied.length / KSFarms.length) * 100
+    );
+
+    const percentGAPCertified = Math.round(
+      (numGAPCertified.length / farms.length) * 100
+    );
+    const percentGAPApplied = Math.round(
+      (numGAPApplied.length / farms.length) * 100
+    );
+    const NSEVPHarvestFarms = recentHarvests.length;
+
+    let totalHarvestsPounds = 0;
+    for (let i = 0; i < totalHarvests.length; i += 1) {
+      totalHarvestsPounds += totalHarvests[i].totalProductionPounds;
+    }
+
+    this.setState({
+      farms,
+      KSFarms,
+      KSGAP,
+      totalHarvests,
+      recentHarvests,
+      GAPCertification,
+      numFarmReferred,
+      numKSGAPAccepted,
+      percentKSGAPCertified,
+      percentKSGAPApplied,
+      percentGAPCertified,
+      NSEVPHarvestFarms,
+      totalHarvestsPounds,
+      percentGAPApplied
+    });
+  }
+
+  getCardStats = isNSEVP => {
+    const {
+      numFarmReferred,
+      numKSGAPAccepted,
+      percentKSGAPCertified,
+      percentKSGAPApplied,
+      percentGAPCertified,
+      NSEVPHarvestFarms,
+      totalHarvestsPounds,
+      percentGAPApplied
+    } = this.state;
     const iconProps = {
       fontSize: 'large',
       style: { color: 'var(--ks-dark-blue)' }
     };
 
+    if (isNSEVP) {
+      return [
+        {
+          icon: <Check {...iconProps} />,
+          name: 'GAP Certification',
+          number: percentGAPCertified,
+          unit: ' %',
+          description: 'of farms in the Group GAP program are GAP Certified'
+        },
+        {
+          icon: <WbSunny {...iconProps} />,
+          name: 'Harvesting Farms',
+          number: NSEVPHarvestFarms,
+          unit: ' farms',
+          description: 'are harvesting this week'
```

Comment:
We made an update to this stat card! You will need to filter for farms in recent harvests with dates in the past 7 days, then remove all duplicate farms (so we are only left with a list of unique farms)

Ask @mylesdomingo about date filtering

![Screen Shot 2021-04-24 at 12 07 52 PM](https://user-images.githubusercontent.com/55935661/115970183-b4ab4c80-a4f5-11eb-9d57-347044340274.png)